### ISSUE TITLE
Pickle error for Python 3.6 and test_peaksFromModelParallel

### ISSUE THREAD

**Issue by matthew-brett** [nipy/dipy#1190]
```
======================================================================
ERROR: dipy.direction.tests.test_peaks.test_peaksFromModelParallel
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/matthew-brett/dipy/venv/lib/python3.6/site-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/home/travis/build/matthew-brett/dipy/venv/lib/python3.6/site-packages/dipy/direction/tests/test_peaks.py", line 509, in test_peaksFromModelParallel
    return_sh=True, parallel=True)
  File "/home/travis/build/matthew-brett/dipy/venv/lib/python3.6/site-packages/dipy/direction/peaks.py", line 419, in peaks_from_model
    nbr_processes)
  File "/home/travis/build/matthew-brett/dipy/venv/lib/python3.6/site-packages/dipy/direction/peaks.py", line 227, in _peaks_from_model_parallel
    repeat(invB)))
  File "/opt/python/3.6.0/lib/python3.6/multiprocessing/pool.py", line 260, in map
    return self._map_async(func, iterable, mapstar, chunksize).get()
  File "/opt/python/3.6.0/lib/python3.6/multiprocessing/pool.py", line 608, in get
    raise self._value
multiprocessing.pool.MaybeEncodingError: Error sending result: '[<dipy.direction.peaks.PeaksAndMetrics object at 0x2974a18>]'. Reason: 'TypeError("can't pickle PeaksAndMetrics objects",)'
```

https://travis-ci.org/matthew-brett/dipy/jobs/208734316#L3259

See #1186 for PR extending testing to Python 3.6.

**Comment by arokem** [comment]
Strangely, I can't replicate this on my laptop. Will try to do this on a linux box.

**Comment by matthew-brett** [comment]
Can you pickle the `PeaksAndMetrics` instances on Python 3.6, Numpy 1.12 ?

**Comment by arokem** [comment]
OK - just tested specifically that. 

No, I can't pickle a PAM object under these conditions. Will continue to investigate.

**Comment by arokem** [comment]
This seems to help: #1195 (but I will admit that I don't entirely understand it)

**Comment by arokem** [comment]
closed through #1195
